runner/config.go

```diff
@@ -334,6 +336,11 @@ func (c *Config) preprocess(args map[string]TomlInfo) error {
 	// CMD will not recognize relative path like ./tmp/server
 	c.Build.Bin, err = filepath.Abs(c.Build.Bin)
 
+	if c.Color.Force == true {
+		// Force colorful output, see https://github.com/fatih/color#disableenable-color
+		color.NoColor = false
+	}
```

Comment:
Usually this kind of configuration flag is done via a string that accepts the following values: auto (default), always, never


```suggestion
	switch c.Color.Mode {
		case "always":
			color.NoColor = false
		case "never":
			color.NoColor = true
		case "auto", "":
			// expected default value
		default:
			return nil, fmt.Errorf("unsupported value for color mode: %s. Expected always, auto, or never", c.Color.Mode)
	}
```

This way people can temporarily disable the color via the configuration.

I know fatih lib supports NO_COLOR, but people expect configuration to control the app without using env

Reply:
LGTM :+1: [`cb701b7` (#768)](https://github.com/air-verse/air/pull/768/commits/cb701b7b319f1ac628a9d97e3954562b8c0bdfa5)